recipes/pygmo/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "1.1.7" %}
+
+package:
+  name: pygmo
+  version: {{ version }}
+
+source:
+  fn: pagmo-{{ version }}.tar.gz
+  url: https://github.com/esa/pagmo/archive/{{ version }}.tar.gz
+  sha256: 6d8fab89ef9d5d5f30f148225bf9b84b2e5a38997f3d68b85547840e9fd95172
+
+build:
+  number: 0
+
+  # https://github.com/esa/pagmo/issues/94
+  skip: true  # [win or osx and py34]
```

Comment:
I see. So that shouldn't be an issue in the feedstock though.

Is it alright if we change it to just `win` then? I know it won't build without error here, but it should build fine in the feedstock where it uses a matrix. Should also save some effort on your end. How does that sound?